components/meta-packages/history/history

```diff
@@ -578,6 +578,7 @@ editor/gedit@2.30.4,5.11-2018.0.0.3
 editor/gedit/gedit-plugins@2.32.0,5.11-2017.0.0.2
 editor/ghex@2.30.0,5.11-2014.0.0.0
 editor/gobby@0.4.94,5.11-2017.0.0.3
+file/rename@2.33.1-2018.0.0.0 file/text-utilities
```

Comment:
To be sure that this package version is always greater than older one, bump branch component  (2018.0.0.1)